flycheck.el

```diff
@@ -3614,7 +3617,11 @@ show the icon."
 
 ;;; Built-in error levels
 (defun flycheck-high-dpi-screen ()
-  "Return t if the user has a HiDPI screen"
+  "Return t if the user has a HiDPI screen."
```

Comment:
Instead of `Return t if …`, ELispers typically write things like `Check whether …`)